rm-datasource/src/main/java/io/seata/rm/datasource/DataSourceProxy.java

```diff
@@ -89,8 +91,15 @@ private void init(DataSource dataSource, String resourceGroupId) {
         } catch (SQLException e) {
             throw new IllegalStateException("can not init dataSource", e);
         }
+        try {
```

Comment:
I think you should remove this code in here, and remove the UndoTableManagerFactory, UndoTableManager, PostgresqlUndoTableManager, finally use script db_undo_log_postgresql.sql instead of it.

Reply:
by the way, please provide the script of db_store_postgre.sql to create global_table, branch_table, lock_table

Reply:
ok, I can use file.conf to load postgresql_***.sql

Reply:
can you check you email address? I send you a email.

Reply:
ok, qq276726581@gmail.com

Reply:
I already send you a email. please check